build/cmake/init.cmake

```diff
@@ -510,7 +510,7 @@ if(wxUSE_GUI)
         if(NOT LIBNOTIFY_FOUND)
             message(WARNING "Libnotify not found, it won't be used for notifications")
             wx_option_force_value(wxUSE_LIBNOTIFY OFF)
-        elseif((LIBNOTIFY_VERSION GREATER 0.7) OR (LIBNOTIFY_VERSION EQUAL 0.7))
+        elseif(LIBNOTIFY_VERSION VERSION_GREATER_EQUAL 0.7)
```

Comment:
`VERSION_GREATER_EQUAL` was added in CMake 3.7 according to the documentation (https://cmake.org/cmake/help/latest/command/if.html#version-greater-equal), but the root-level CMake has the minimum compatibility version set to 2.8.12, so there is the possibility that this will not be available and error.

Reply:
Thanks for noticing. I'll test with some older CMake versions and see if we can still support 2.8.12 or if we should use a different minimum version.
3.7 still works after a small change.

Reply:
It does not work with `2.8.12`, and it never did. Because `2.8.12` does not support the `VERSION` parameter in
https://github.com/wxWidgets/wxWidgets/blob/d72a3f7cecd1df5f1da946e11ad7ada7b3edba42/CMakeLists.txt#L96
I'll change it to `3.0`, that seems to work.

Reply:
https://github.com/wxWidgets/wxWidgets/pull/22312